src/cie/README.md

```diff
@@ -0,0 +1,6 @@
+# CIE
+
+This library provides a components and a set of utilities to interact with the physical [CIE (Carta d'Identità Elettronica)](https://www.cartaidentita.interno.gov.it/) card. It can be used to [obtain a eID](../credential/issuance/README.md) via strong authentication.
```

Comment:
```suggestion
This library provides a components and a set of utilities to interact with the physical [CIE (Carta d'Identità Elettronica)](https://www.cartaidentita.interno.gov.it/) card. It can be used to [obtain an eID](../credential/issuance/README.md) via strong authentication.
```